docs/source/guide/explanation/additional_features/tiling.rst

```diff
@@ -0,0 +1,148 @@
+    The below context will be provided during evaluation:
+
+    .. code-block:: 
+
+        [>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>] 650/650, 17.2 task/s, elapsed: 38s, ETA:     0s
+        ==== merge: 7.326097726821899 sec ====
+
+
+Enable Tiling via OTX Training CLI 
+==================================
+
+Currently, tiling is supported for both detection and instance segmentation models. Please refer to :doc:`../algorithms/object_detection/object_detection` and :doc:`../algorithms/segmentation/instance_segmentation` for more details.
+
+To enable tiling in OTX training, set ``tiling_parameters.enable_tiling`` parameter to 1. Here's an example of enabling tiling for the SSD model template:
+
+.. code-block::
+
+    otx train Custom_Object_Detection_Gen3_SSD --train-data-roots tests/assets/small_objects --val-data-roots tests/assets/small_objects params --tiling_parameters.enable_tiling 1
+
+.. note::
+
+    To learn how to deploy the trained model and run the exported demo, refer to :doc:`../../tutorials/base/deploy`.
+
+    To learn how to run the demo in CLI and visualize results, refer to :doc:`../../tutorials/base/demo`.
+
+
+Tile Size and Tile Overlap Optimization
+-----------------------------------------
+By default, the OpenVINO Training Extensions automatically optimize tile size and tile overlap to ensure efficient execution without compromising accuracy.
+
+To strike a balance between patch size and computational efficiency, the OpenVINO Training Extensions incorporate adaptive tiling parameter optimization. These features enable the proper tuning of tile size and other tiling-related parameters to ensure efficient execution without compromising accuracy.
+
+Adaptive tiling parameter optimization works by finding the average object size in the training dataset and using that to determine the tile size. Currently, the average object size to tile size ratio is set to 3%. For example, if the average object size is 100x100 pixels, the tile size will be around 577x577 pixels.
+
+This computation is performed by dividing the average object size by the desired object size ratio (default: 3%) and then taking the square root. This ensures that the objects are large enough to be detected by the model. The object size to tile size ratio can also be configured with ``tiling_parameters.object_tile_ratio`` parameter. 
+
+Here's an example of setting the object size ratio to 5%:
+
+.. code-block:: 
+    
+    otx train Custom_Object_Detection_Gen3_SSD --train-data-roots tests/assets/small_objects --val-data-roots tests/assets/small_objects
+    params --tiling_parameters.enable_tiling 1          \  # enable tiling
+           --tiling_parameters.enable_adaptive_params 1 \  # enable automatic tiling parameter optimization
+           --tiling_parameters.object_tile_ratio 0.05   \  # set the object size ratio to 5%
```

Comment:
If we'd use workspace creation how we do in most of the tutorials now, we'd make this line shorter:

    otx train \
    params --tiling_parameters.enable_tiling 1          \  # enable tiling
           --tiling_parameters.enable_adaptive_params 1 \  # enable automatic tiling parameter optimization
           --tiling_parameters.object_tile_ratio 0.05   \  # set the object size ratio to 5%

Reply:
But if we'd keep such example, it's better to reformat it a little:

```
 otx train Custom_Object_Detection_Gen3_SSD 
    --train-data-roots tests/assets/small_objects \
    --val-data-roots tests/assets/small_objects \
    params --tiling_parameters.enable_tiling 1          \  # enable tiling
           --tiling_parameters.enable_adaptive_params 1 \  # enable automatic tiling parameter optimization
           --tiling_parameters.object_tile_ratio 0.05   \  # set the object size ratio to 5%
```